app/src/main/java/org/simple/clinic/appconfig/AppConfigRepository.kt

```diff
@@ -32,6 +33,10 @@ class AppConfigRepository @Inject constructor(
     return selectedCountryPreference.asObservable()
   }
 
+  fun currentDeployment(): Deployment? {
+    return selectedDeployment.get().toNullable()
```

Comment:
In which would deployment be nullable? Oh is it because the old versions may not have deployments?

Reply:
The existing preferences don't contain deployments. So it will be nullable when we try to get it.